chpl/chpl-api/e2e/collections/testing-lab-controller.postman_collection.json

```diff
@@ -1,6 +1,6 @@
 {
 	"info": {
-		"name": "testing-lab-controller Copy",
+		"name": "testing-lab-controller",
```

Comment:
This is in Ashwini's PR already so please remove

Reply:
Whether it's in a different PR or not, it's still valid for it to be in this one too. Whichever gets to staging first will mean that the other PR wouldn't need this change, but it should be okay for it here